py_experimenter/database_connector_mysql.py

```diff
@@ -13,13 +13,13 @@
 class DatabaseConnectorMYSQL(DatabaseConnector):
     _prepared_statement_placeholder = '%s'
 
-    def __init__(self, experiment_configuration: ConfigParser, use_codecarbon:bool, codecarbon_config:ConfigParser, database_credential_file_path:str):
+    def __init__(self, experiment_configuration: ConfigParser, use_codecarbon:bool, codecarbon_config:ConfigParser, database_credential_file_path:str, logger_name:str = 'py_experimenter'):
```

Comment:
Default: `PyExperimenter`

Actually I am wondering if we really need the default here? Isn't it enough to have it in the constructor of `PyExperimenter`

Reply:
You're correct, i fixed that